scripts/lib/install

```diff
@@ -99,9 +99,19 @@ export LANG="en_US.UTF-8"
 export LANGUAGE="en_US.UTF-8"
 
 # Check for a supported OS release.
-apt-get install -y lsb-release
-os_info="$(lsb_release --short --id --release --codename)"
-{ read -r os_id; read -r os_release; read -r os_codename; } <<< "$os_info"
+if [ -f /etc/debian_version ]; then
+    apt-get install -y lsb-release
+    os_info="$(lsb_release --short --id --release --codename)"
+    { read -r os_id; read -r os_release; read -r os_codename; } <<< "$os_info"
+elif [ -f /etc/os-release ]; then
+    source /etc/os-release
+    export os_id="$ID"
+    export os_release="$VERSION_ID"
+    # CentOS 7 doesn't have VERSION_CODENAME
+    export os_codename="$VERSION_ID"
```

Comment:
Like we did with the development provision scripts, I’d prefer to stay aligned with upstream terminology: `os_id`, `os_version_id`, no `os_codename` for CentOS at all.

Reply:
Fixed.